xbmc/dialogs/GUIDialogKeyboardGeneric.cpp

```diff
@@ -34,13 +35,7 @@
 #include <mutex>
 
 #ifdef TARGET_ANDROID
-#include <androidjni/Intent.h>
-#include <androidjni/RecognizerIntent.h>
-#include <androidjni/ArrayList.h>
 #include "platform/android/activity/XBMCApp.h"
-
-#define ACTION_RECOGNIZE_SPEECH_REQID 543
```

Comment:
Prefer constants over defines, because of type safety:

```c++
namespace
{
constexpr int ACTION_RECOGNIZE_SPEECH_REQID = 543;
}

Reply:
Hello, I've deleted this define because I think it's not needed

Reply:
Even better.